api/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/api/tilgang/TilgangProducer.kt

```diff
@@ -23,8 +23,7 @@ class TilgangProducer(private val rapid: RapidsConnection) {
             Key.IDENTITETSNUMMER to fnr.toJson(),
             Key.FORESPOERSEL_ID to forespørselId.toJson(),
             Key.BOOMERANG to mapOf(
-                Key.NESTE_BEHOV.str to listOf(BehovType.TILGANGSKONTROLL).toJson(BehovType.serializer()),
-                Key.INITIATE_ID.str to initiateId.toJson()
```

Comment:
Er det noen grunn til at initiate_id fjernes her, men beholdes i TrengerProducer? (mtp konsistens)

Reply:
Hmm... jeg hadde en feil merge som skjedde, kanskje noe gammel har leaked in. Bra at du så det. Også 
Jeg vil endre det til:

 rapid.publish(
            Key.EVENT_NAME to EventName.HENT_PREUTFYLT.name.toJson(),
            Key.BEHOV to listOf(BehovType.HENT_IM_ORGNR).toJson(BehovType.serializer()),
            Key.NESTE_BEHOV to listOf(BehovType.TILGANGSKONTROLL).toJson(BehovType.serializer()),
            Key.UUID to initiateId.toJson(),
            Key.IDENTITETSNUMMER to fnr.toJson(),
            Key.FORESPOERSEL_ID to forespørselId.toJson()
        )

Reply:
Det er ikke noe Helsebro i bildet her så tror jeg vi ikke tremger Boomerang.